subject_files_status_for_dpdash.py

```diff
@@ -35,25 +35,25 @@ def _latest_mtime(p: Path) -> str:
     return datetime.fromtimestamp(latest).strftime('%Y-%m-%d')
 
 
-def check_upenn_cnb(subject_raw_path: Path) -> bool:
+def check_upenn_cnb(subject_raw_path: Path) -> int:
     '''Check if there are SPLLT-A and NOSPLLT in the Penn CNB json file'''
     json_paths = list((subject_raw_path / 'surveys').glob('*UPENN.json'))
 
     if len(json_paths) == 0:  # No file
-        return False
+        return 0
 
     with open(json_paths[0], 'r') as fp:
         data = json.load(fp)
 
     if len(data) != 2:
-        return False
+        return 0
 
     session_batteries = [x['session_battery'] for x in data]
     spllta_check = 'SPLLT-A' in session_batteries
     nospllt_check = ('ProNET_NOSPLLT' in session_batteries or
                      'PRESCIENT_NOSPLLT_C1' in session_batteries)
 
-    return spllta_check & nospllt_check
+    return spllta_check * nospllt_check
```

Comment:
This is a very good design. I didn't know the `*` casts them to integer directly.